### ISSUE TITLE
Questions about training and hyperparameter

### ISSUE THREAD

**Issue by glorioushonor** [YuliangXiu/ICON#201]
Hello, I would like to ask you about the setting of the hyperparameter. After training ICON sevaral times, I found that the best saved model was in the middle of the first epoch (see figure). I'm not sure whether that's correct.But I found that my quantitative results were all within 1cm, which again seemed correct.
![image](https://user-images.githubusercontent.com/120625228/232077961-01fd096c-d618-4e9b-a51a-243857e65e68.png)

Perhaps because the learning-rate is set too high and the convergence speed of ICON model is too fast, it is difficult to find the best model. It can also be seen from the loss chart that loss begins to fluctuate violently after a sharp decline.
![image](https://user-images.githubusercontent.com/120625228/232078352-15f6b18d-2407-4844-bcd2-53481b4d3830.png)
